contrib/plasma-workspace/template.py

```diff
@@ -132,6 +136,41 @@ def post_install(self):
 
     self.rm(self.destdir / "usr/lib/systemd/user", recursive=True)
 
+    self.mkdir(
+        self.destdir / "usr/share/wallpapers/Chimera/contents/images",
+        parents=True,
+    )
+    self.mkdir(
+        self.destdir / "usr/share/wallpapers/Chimera/contents/images_dark"
+    )
+    self.install_file(
+        self.files_path / "metadata.json", "usr/share/wallpapers/Chimera"
+    )
+    self.install_link(
+        "usr/share/wallpapers/Chimera/contents/images/2560x1600.svg",
+        "/usr/share/backgrounds/chimera/bg-l.svg",
+        absolute=True,
+    )
+    self.install_link(
+        "usr/share/wallpapers/Chimera/contents/images_dark/2560x1600.svg",
+        "/usr/share/backgrounds/chimera/bg-d.svg",
+        absolute=True,
+    )
```

Comment:
using symlinks here feels like a hack (also because of the `broken_symlinks` thing above, but idk

also should this be moved into a subpackage? or maybe even into chimera-artwork itself? it's just metadata.json and a couple symlinks so i think putting it into chimera-artwork is reasonable

Reply:
doesn't have to be absolute i think